### PR TITLE
Geospatial Data Support - Frontend

### PR DESCRIPTION
**Pull request by dilallkx**
**CHANGES:**
- Frontend:
  - New dependency on _leaflet-geoman-free_ package
  - Network element configuration support for location, path, radius, EOP mode & velocity
  - Configuration capabilities for UE, POA & Compute node location using map clicks
  - Configuration capabilities for UE path using map clicks
  - Map attributions
  - Default zoom levels & center set to 2 & [0,0]
  - New event automation configuration using frontend checkboxes

**TESTING:**
- Cypress & UT pass
- Manually verified element configuration using maps
- Manually tested automation endpoint configuration using frontend

### CODE REVIEW

File: js-apps/meep-frontend/src/js/containers/idc-map.js [hunk 1]
```diff
@@ -128,6 +253,13 @@ class IDCMap extends Component {
       'compute': this.computeOverlay
     };
     
+    // // Set z-index
```
Comment (roymx) [inline js-apps/meep-frontend/src/js/containers/idc-map.js:256]: Dead code to remove?

File: js-apps/meep-frontend/src/js/containers/idc-map.js [hunk 2]
```diff
@@ -193,7 +344,11 @@ class IDCMap extends Component {
     });
 
     if (existingMarker === undefined) {
-      // Create new UE marker & path
+      // Create path, if any
+      // var p = ue.path ? L.GeoJSON.geometryToLayer(ue.path) : null;
+      var p = pathLatLngs ? L.polyline(pathLatLngs) : null;
+
```
Comment (roymx) [inline js-apps/meep-frontend/src/js/containers/idc-map.js:350]: Shouldn't we set the UE asset name in the path options?

File: js-apps/meep-frontend/src/js/containers/idc-map.js (hunk 1 above)
Comment (dilallkx) [reply js-apps/meep-frontend/src/js/containers/idc-map.js]: Removed.

File: js-apps/meep-frontend/src/js/containers/idc-map.js (hunk 2 above)
Comment (dilallkx) [reply js-apps/meep-frontend/src/js/containers/idc-map.js:343]: Added as suggested.

**Pull request merged**